uqcsbot/scripts/dominos.py

```diff
@@ -7,40 +7,43 @@
 from uqcsbot.utils.command_utils import loading_status, UsageSyntaxException
 import requests
 
-MAX_COUPONS = 10 # Prevents abuse
+MAX_COUPONS = 10  # Prevents abuse
 COUPONESE_DOMINOS_URL = 'https://www.couponese.com/store/dominos.com.au/'
 
+
 class Coupon:
     def __init__(self, code: str, expiry_date: str, description: str) -> None:
         self.code = code
         self.expiry_date = expiry_date
         self.description = description
-    
+
     def is_valid(self) -> bool:
         try:
             expiry_date = datetime.strptime(self.expiry_date, '%Y-%m-%d')
             now = datetime.now()
-            return expiry_date.year >= now.year and expiry_date.month >= now.month and\
-                expiry_date.day >= now.day
-        except:
+            return (expiry_date.year >= now.year and expiry_date.month >= now.month and
+                    expiry_date.day >= now.day)
+        except Exception:
             return True
 
     def keyword_matches(self, keyword: str) -> bool:
         return keyword.lower() in self.description.lower()
 
+
 @bot.on_command("dominos")
 @loading_status
 def handle_dominos(command: Command):
-    '''
-    `!dominos [--num] N [--expiry] <KEYWORDS>` - Returns a list of dominos coupons (default: 5 | max: 10)
-    '''
+    """
+    `!dominos [--num] N [--expiry] <KEYWORDS>`
+    Returns a list of dominos coupons (default: 5 | max: 10)
+    """
     command_args = command.arg.split() if command.has_arg() else []
 
     parser = argparse.ArgumentParser()
 
     def usage_error(*args, **kwargs):
         raise UsageSyntaxException()
-    parser.error = usage_error # type: ignore
+    parser.error = usage_error  # type: ignore
```

Comment:
Is there a reason for this change?

Reply:
`flake8` gives [E261](https://lintlyci.github.io/Flake8Rules/rules/E261.html).